### PR TITLE
feat: standard banners

### PR DESCRIPTION
**Pull request by helloiambguedes**
## Summary

TBD

## Notion link

TBD

## How to test?

TBD

## Notes

TBD

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "rules": {
+    "eol-last": 2
+  },
+  "parser": "@babel/eslint-parser",
+  "env": {
+    "es6": true,
+    "node": true,
+    "browser": true
+  }
+}
```
Comment (verdefred) [inline .eslintrc.json:15]: can you add a rule to add a new empty line at the end of every file?

**Review by verdefred — COMMENTED** [review]
Great job, just left a small comment

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (verdefred) [inline .eslintignore:1]: this one is still missing the empty line 😄

File: app/packs/src/components/talent/Edit/LaunchTokenModals.jsx [hunk 3]
```diff
@@ -348,7 +348,7 @@ const LaunchTokenModals = (props) => {
     if (result) {
       setFactory(newOnChain);
     } else {
-      setDeploy("Unable to deploy token");
+      setDeploying("Unable to deploy token");
```
Comment (francisco-leal) [inline app/packs/src/components/talent/Edit/LaunchTokenModals.jsx:351]: I don't think you can make this change. Deploying is expected to be a boolean. If it isn't used then just remove it and add a toast error for example

File: app/packs/src/components/discovery/discovery_row.jsx [hunk 4]
```diff
@@ -7,6 +7,7 @@ import NewTalentCard from "src/components/design_system/cards/NewTalentCard";
 import Button from "src/components/design_system/button";
 import Tag from "src/components/design_system/tag";
 import { displayableAmount } from "src/utils/viewHelpers";
+import { destroy } from "../../utils/requests";
```
Comment (francisco-leal) [inline app/packs/src/components/discovery/discovery_row.jsx:10]: We already have `import { get, post } from "src/utils/requests";` a couple lines below, can you add the destroy import there?

**Review by francisco-leal — APPROVED** [review]

**Pull request merged**